webcam/public/index.html

```diff
@@ -0,0 +1,77 @@
+
+
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8">
+    <title></title>
+  </head>
+  <body>
+    Name:<input type="text" id="Name" />
+    Comment:<input type="text" id="Comment" />
+  <!--<input type="submit" id="button" />-->
+    Picture:<input type="submit" value="take picture" id = "click"/>
```

Comment:
why is value included here?

Reply:
to change my text of button to take picture. i guess value is the text what you want to print on the button

Reply:
Dude you have two index.html